src/reducers/photos.js

```diff
@@ -0,0 +1,15 @@
+import { ACTION_TYPES } from '../actions/index.js';
+
+const INITIAL_STATE = {
+  list: [],
+};
+
+// Reducer
+export const photos = (state = INITIAL_STATE, {type, payload}) => {
+  switch (type) {
+  case ACTION_TYPES.getPhotos:
+    return {...state, ...{list: payload.photos}};
```

Comment:
I was wrong about this^. It's actually an example of [object spread operator](http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html) use.